index.rst

```diff
@@ -1,54 +1,148 @@
+
+In :numref:`figure-2020031300131` we show visit ``2020031300131`` with rulers showing the arc length and its distance to the apparent center of rotation.
+The measurement values are :math:`0.003\deg` and :math:`0.062\deg`, which translates to an angle of :math:`2.77\deg`.
+If we compute the variation in the rotator position from the start to finish of that exposure, we obtain a variation of :math:`2.14\deg`, which is consistent with what we get from the arc-length above.
+
+.. figure:: /_static/HD98993_131.png
+   :name: figure-2020031300131
+   :target: ../_images/HD98993_131.png
+   :alt: HD 98993 390s exposure
+
+   A long (390s) exposure of HD 98993 that clearly shows the rotator direction issue.
+
+:numref:`figure-stackall` shows an inset of a stack of the three images.
+The images are combined in pixel space to highlight the effect and arbitrarily scaled to highlight the effect.
+The reference star from the first and last images are marked for reference along with the arc from the long exposure.
+It is quite clear that the images rotates with the expected rate as if the rotator was moving in the wrong direction.
+
+.. figure:: /_static/HD98993_stack.png
+   :name: figure-stackall
+   :target: ../_images/HD98993_stack.png
+   :alt: HD 98993 stack
+
+   Stacked images of HD 98993 showing the effect of rotation over the three test images, a short 5s exposure followed by long a 390s and a 30s exposure.
+   The images are arbitrarily scaled to highlight the effect.
+
+
+.. _section-HD68450:
+
+HD 68450
+--------
+
+After performing the analysis on :ref:`HD 98993 <section-HD98993>` and we are confident that the rotator is moving in the wrong direction, we should be able to verify it using observations of different targets at different times with different requested sky rotation angles.
+
+The analysis shown here was repeated in a number of different targets but we choose ``HD 68450`` because it is easy to identify the field rotation.
+In :numref:`figure-HD68450` we show a finding chart with the field orientation (direction of North and East axis) and the images taken with the Auxiliary Telescope on the left and right-hand side, respectively.
+Visits ``2020031200197`` and ``2020031200208`` are shown at the top and bottom, respectively, with two different orientations.
+Images in the middle (labeled "original orientation") are shown with the sky angle taken from the commanded position.
+The right-most images (labeled "computed orientation") are shown with the sky angle computed from the data available in the EFD.
+The calculation is done by taking the time when the exposure starts and ends and querying for the telescope azimuth, elevation and nasmyth angle.
+Then, assuming the rotator has the reversed direction, we compute the expected sky angle.
+It is clear, from visual inspection, that the computed sky angle should be close to the correct sky angle orientation.
```

Comment:
It is clear, from visual inspection, that the computed sky angle now matches the correct sky angle orientation seen in th finding chart.